GetandTake/Core/Utilities/Interceptors/AspectInterceptorSelector.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Castle.DynamicProxy;
+using System.Reflection;
+
+namespace GetandTake.Core.Utilities.Interceptors;
+
+public class AspectInterceptorSelector : IInterceptorSelector
+{
+    public IInterceptor[] SelectInterceptors(Type type, MethodInfo method, IInterceptor[] interceptors)
+    {
+        var classAttributes = type.GetCustomAttributes<MethodInterceptionBaseAttribute>
+            (true).ToList();
```

Comment:
Let's use named parameters instead of hardcoding values. For some devs, not familiar with code base or 3rd party solutions it will be difficult to understand the purpose of this value